src/Lightning.ts

```diff
@@ -179,15 +185,15 @@ export const LightningMixin = (superclass) => class extends superclass {
       throw new RouteFindingError(undefined, {logger: lightningLogger, probingSuccess: false, success: false})
     }
 
-    const value = JSON.stringify(route)
+    const value = JSON.stringify({...route, pubkey})
     await redis.set(key, value, 'EX', 60 * 5); // expires after 5 minutes
 
     lightningLogger.info({ redis: { key, value }, probingSuccess: true, success: true }, "successfully found a route")
     return route.fee
   }
 
   // FIXME this should be static
-  async validate(params: IFeeRequest, lightningLogger) {
+  async validate({params, logger}: {params: IFeeRequest, logger: Logger}) {
```

Comment:
Shifting this function to `lndUtils.ts` might make sense

Reply:
yup